README.md

```diff
@@ -176,41 +187,60 @@ $ git commit -m "Initial commit"
 ```
 
 We can see that Git has created a new version of our repo, represented by the
-_SHA_ `e55477d`. SHAs are the identification system that `git` uses to keep
-track of versions; they're long complex numbers and letters that are unlikely to
-be duplicated. If you're following along, the SHA you'll see in your terminal
-will be different.
+_SHA_ (Secure Hash Algorithm) `e55477d`. SHAs are the identification system that
+`git` uses to keep track of versions; they're long complex numbers and letters
+that are unlikely to be duplicated (the value shown above is actually a
+shortened version of the full SHA). If you're following along, the SHA you'll
+see in your terminal will be different.
 
 The `commit` command committed 1 file.
 
-Now, if we type `git status`, we'll see that it is at a "clean state", and there
-is nothing to commit and no new changes.
+Now, if we type `git status`, we'll see that it is at a "clean state": there are
+no new changes which means there's nothing to commit.
 
 ```console
 $ git status
 On branch main
 nothing to commit, working tree clean
 ```
 
-If we were to make another change, for example, to README.md, we could then add
-another commit with this new set of changes with
+We won't do this just yet, but if we were to open the README.md file in our text
+editor and add some text to it, we could then add our file and commit this new
+set of changes with the following:
+
+```console
+$ git add README.md
+$ git commit -m "Updates README.md"
+```
+
+There are also a couple of shortcuts we can use:
+
+```console
+$ git add .
+```
+
+This command adds _all_ the files that have been changed since the last commit
+(in this case, just `README.md`).
+
+Or we can combine the two steps of adding and committing our file into a single
+command:
 
 ```console
 $ git commit -am "Updates README.md"
 ```
 
-The use of the `-am` flag here is a shortcut that combines the steps of adding
-changed files and committing them. The `-a` flag tells git to add 'all changes',
-i.e., all files that have been changed since the last commit. The `-m` flag,
-like before, tells git that we want to specify a commit message, in this case,
-`"Updates README.md"`. To review some of your Unix CLI skills, this could also
-be written as:
+Here we're combining the `a` and `m` flags. As with the `add .` shortcut, the
+`a` flag tells git to add 'all changes', i.e., all files that have been changed
```

Comment:
Important distinction here: `a` will commit changes to all **tracked** files, not all files in the directory. So if a user creates a new file and then uses `git commit -a` the new file won't be staged.

Reply:
@ihollander Does this look ok?

```
Here we're combining the `a` and `m` flags. As with the `add .` shortcut, the
`a` flag tells git to add 'all changes', i.e., all files that have been changed
since the last commit. Note, however, that using the `a` flag will only commit
files that are _already being tracked_; if you've created a _new_ file, you'll
need to stage it first using `git add`.

The `-m` flag, like before, tells git that we want to specify a commit message,
in this case, `"Updates README.md"`. This command could also be written as:

...
```

Reply:
Yep, looks great! 👍

Reply:
👍